src/illuminator/engine.py

```diff
@@ -10,6 +10,7 @@
 from mosaik.scenario import World as MosaikWorld
 from datetime import datetime
 
+current_model = {}
```

Comment:
Right now we are using this as a global variable. I do not like this and if we are not pressed on time, we should fix this.